common/config/config.go

```diff
@@ -262,8 +262,10 @@ type (
 		EnableGlobalDomain bool `yaml:"enableGlobalDomain"`
 		// FailoverVersionIncrement is the increment of each cluster version when failover happens
 		FailoverVersionIncrement int64 `yaml:"failoverVersionIncrement"`
-		// MasterClusterName is the master cluster name, only the master cluster can register / update domain
+		// PrimaryClusterName is the primary cluster name, only the primary cluster can register / update domain
 		// all clusters can do domain failover
+		PrimaryClusterName string `yaml:"primaryClusterName"`
```

Comment:
Just a convention that we introduced breaking changes in configuration -- can we have a version being compatible with warning logs? Likely 0.22.0. And then in 0.23.0 we will remove `MasterClusterName` completely.

Reply:
Yes. This versio is backward compatible. Added a logging message.